Automattic/wp-calypso
# Multi-Domains: Only one primary CTA when domains are in cart

Related to https://github.com/Automattic/dotcom-forge/issues/4140

## Proposed Changes

When domain(s) is in the cart, the Primary CTA should only be the Continue. All the rest of Select should be secondary CTA’s

| Before | After |
| --- | --- |
| ![image](https://github.com/Automattic/wp-calypso/assets/402286/b8e30bad-03e1-4f0f-a3c5-88cfa101b9cf) | ![image](https://github.com/Automattic/wp-calypso/assets/402286/64d1595c-dad7-4244-989b-19d600bbf977) |

## Testing Instructions

* Go to `/start/domains?flags=domains/add-multiple-domains-to-cart`
* Add a not-featured domain to the cart
* The featured domains should not show as primary buttons
